.github/workflows/main.yaml

```diff
@@ -33,7 +33,9 @@ jobs:
       - name: Run tests
         env:
           TEST_API_KEY_WRITE: ${{ secrets.TEST_API_KEY_WRITE }}
-          TEST_API_KEY_READ: ${{ secrets.TEST_API_KEY_READ }}
+          TEST_OIDC_READ_CLIENT_ID: ${{ secrets.TEST_OIDC_READ_CLIENT_ID }}
+          TEST_OIDC_READ_CLIENT_SECRET: ${{ secrets.TEST_OIDC_READ_CLIENT_SECRET }}
+          TEST_OIDC_READ_TENANT: ${{ secrets.TEST_OIDC_READ_TENANT }}
```

Comment:
Isn't this the same as bluefield oidc credentials? but maybe it is good to have redundant env vars due to naming.